### DISCUSSION TITLE
How to find the --strip-unknown-mods option -

### DISCUSSION THREAD

**Discussion by BenoitBe** [vdemichev/DiaNN#188] [Q&A]
Hello,

I am trying to use the soft, but we perform samples prep using GASP protocol resulting of C propioamide (UNIMOD 24) in addition to other modifications, depending the project. I want to create the spectral library using a fasta proteome from Uniprot.

In the paragraph "Creation of spectral librairies", I have shown this: If you would like to perform prediction for peptides bearing some other modification(s), can try the --strip-unknown-mods option - then DIA-NN will ignore all the modifications not yet supported by the deep learning predictor.

But I am not able to find the --strip-unknown-mods option -. Maybe someone can help me?

Thanks in advance

**Comment by vdemichev** [comment]
Hi Benoit,

Type "--strip-unknown-mods" (without quotes) in the 'Additional options' textbox.

Best,
Vadim

**Comment by BenoitBe** [comment]
Ok, thanks a lot :)
